lindera/src/token_filter/japanese_compound_word.rs

```diff
@@ -144,6 +145,10 @@ impl TokenFilter for JapaneseCompoundWordTokenFilter {
                 }
             }
         }
+        if compound_token_opt.is_some() {
```

Comment:
Where is the test case for this change?

Reply:
Added a test for JapaneseCompoundWordTokenFilter. Thanks.